timetrack-cli.cabal

```diff
@@ -58,6 +58,7 @@ executable timetrack
                      , RankNTypes
                      , ScopedTypeVariables
                      , StandaloneDeriving
+                     , TemplateHaskell
```

Comment:
stylish-haskell is acting a fool... not sure if this is the answer, but it's complaining less than before